<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ui2_myPokemon">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>查看精灵</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui2_allPlayers">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>查看玩家</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui2_saveAndQuit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>退出游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui2_returnSignIn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>返回登录界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui4_returnMainUi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui3_returnMainUi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_background">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ui1_signUp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>注册并登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui1_password">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ui1_signIn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ui1_userName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>201</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ui1_label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
